pkg/backup/backup/backup_manager.go

```diff
@@ -244,7 +244,7 @@ func (bm *backupManager) checkVolumeBackupInitializeJobRunning(backup *v1alpha1.
 		// all the volume snapshots has created
 		return nil
 	}
-	if !v1alpha1.IsVolumeBackupInitialized(backup) || v1alpha1.IsVolumeBackupInitializeFailed(backup) {
+	if v1alpha1.IsVolumeBackupInitializeFailed(backup) {
```

Comment:
Noob question. checkVolumeBackupInitializeJobRunning is used to detect init job failures. Based on the "err" returned from this function. IF IsVolumeBackupInitializeFailed == true, then why we return nil, rather than returning an error.

Reply:
Because the aim of this function is to detect init job failure and set backup CR to `IsVolumeBackupInitializeFailed`. So if `IsVolumeBackupInitializeFailed == true`, it means we don't need this check. If we return error here, it can block the main logic of the backup controller.